app/uk/gov/hmrc/eeittadminfrontend/connectors/GithubConnector.scala

```diff
@@ -38,7 +38,7 @@ class GithubConnector(authorization: Authorization, maybeProxy: Option[Proxy], w
 
   val repoOwner = authorization.repoOwner
   val repoName = authorization.repoName
-  val master = Some("master")
+  val main = Some("main")
```

Comment:
We first need to make sure `https://github.com/hmrc/gform-templates` is migrated to main, before we merge this.

Reply:
`https://github.com/hmrc/gform-templates` has been migrated to `main`.